.github/workflows/appimage.yml

```
@@ -0,0 +1,74 @@
+name: Build MG AppImage
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version number'
+        required: false
+        type: string
+  push:
+    branches: develop
+  pull_request:
+    branches: develop
+    paths:
+      - '**appimage.yml'
+
+jobs:
+  build-mg-appimage:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ inputs.version }}
+    strategy:
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
+
+    steps:
+      - name: Checkout MegaGlest
+        uses: actions/checkout@v4
+
+      - name: Checkout Data
+        uses: actions/checkout@v4
+        with:
+          repository: "megaglest/megaglest-data"
+          path: "data/glest_game"
+
+
+      - if: ${{ ! contains(matrix.platform, 'amd64') }}
+        uses: docker/setup-qemu-action@v3
```

```suggestion
        uses: docker/setup-qemu-action@v3.2.0
```

Note 'v3' points to v3.3.0 https://github.com/docker/setup-qemu-action/tags This is that standard way github actions are released. When minor versions are released, the major tag is updated on the action repo.